model/sources/IDU/README.md

```diff
@@ -0,0 +1,122 @@
+# IDU
+
+The IDU(Internal Displacements Updates) tracks the number of the people who are internally displaced in a region or a country due to `disaster` or `conflict`.
```

Comment:
Should we mention here that we only deal with the displacement due to `disaster` data at the moment?

Reply:
sure